PeerioTechnologies/peerio-desktop
# fix: positioning of elements when adding user to existing room

#### Relevant info and issue/PR links
https://app.clubhouse.io/peerio/story/17383/desktop-search-field-in-add-member-to-room-view-broken

#### Testing instructions
Check that styling is fixed in add-user-to-existing-room screen, and that styling is unaffected in new room screen.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
